src/Microsoft.VisualStudio.Editors/Microsoft.VisualStudio.Editors.pkgdef

```diff
@@ -1,3 +1,4 @@
+; TODO generate this file https://github.com/dotnet/project-system/issues/68
```

Comment:
This is the comment syntax?

Reply:
From https://github.com/madskristensen/PkgdefLanguage:

![image](https://user-images.githubusercontent.com/350947/61624866-c9bdd080-acbc-11e9-8781-b72da960a3dc.png)